### ISSUE TITLE
Improve MangaFox page loading

### ISSUE THREAD

**Issue by DakuTree** [DakuTree/manga-tracker#84]
MangaFox is one of the few sites we're tracking that has some form of bot protection when loading actual image pages. If we try and load the pages too quickly, it throws an error.  
This isn't a problem if you're just loading one chapter, but if you try and open multiple it often causes a lot of pages to fail.

I noticed earlier today that MF pages actually include the image for both the current AND next page, which means we could effectively reduce our AJAX calls in half, and load the chapter quicker.  
The bot protection only applies to the web pages, not the images.

The main problem with all this is, we can't support this with our current setup. We assume each page we check will only return one page, not two. Even if we just check every second page, we run into the issue where if the page we check fails, we only get an error container for that page and not the next one.  

Basically, we'd need a small rewrite of how setupViewer handles page loading to make this work properly. Still trying to consider the best way to do this though.

Just for my own later reference, this works with grabbing the next img.
```javascript
let following_image = $(data.replace(/^[\s\S]*(<div style="display:none">[\s\S]*<\/div>)[\s\S]*<\/body>[\s\S]*$/, '$1')).find("img:last");
if(following_image.length) {
_this.setupViewerContainer($(following_image).attr('src'), this.page + 1);
}
```

**Comment by DakuTree** [comment]
Alternate solution here is using the mobile "all on one page" loader. (m.mangafox.me/roll_manga/...)  
From what I've looked at, this appears to load exactly the same images as the desktop viewer. If this is the case, we could just grab this and use the custom image list.
